Dockerfile

```diff
@@ -2,8 +2,12 @@
 ARG BUILDERIMAGE="golang:1.17"
 
 ARG ERASERBASEIMAGE="gcr.io/distroless/static:latest"
+ARG COLLECTORBASEIMAGE="gcr.io/distroless/static:latest"
```

Comment:
looks like we are going to be adding same image in scanner too. can we just call these `STATICBASEIMAGE` and `STATICNONROOTBASEIMAGE`